cypress-tests/cypress/e2e/PaymentTest/00013-SaveCardFlow.cy.js

```diff
@@ -312,7 +312,7 @@ describe("Card - SaveCard payment flow test", () => {
           "card_pm"
         ]["SaveCardUseNo3DSAutoCaptureOffSession"];
```

Comment:
this config does not exist in `WorldPay.js` either. So, it will be fetched from `commons.js` and by default, it will expect a 5xx or 4xx.

Reply:
Yes! that's the expectation at the moment (since mandates are not implemented yet)